changelogs/template.yml

```diff
@@ -25,7 +25,8 @@
 #- (Patternlab) [Header] DP-4562: Set focus state for search on mobile menu in mobileNav module #473
```

Comment:
you can remove these two lines https://github.com/massgov/mayflower/pull/812/commits/0c6db715303a56d8585a633cace3c7a1c2def37a#diff-0de3b49620e59f1d729d4ca54219d53dR25-R26